src/Symfony/Component/String/Tests/AbstractUnicodeTestCase.php

```diff
@@ -751,7 +751,7 @@ public static function provideReverse()
             [
                 ['äuß⭐erst', 'tsre⭐ßuä'],
                 ['漢字ーユニコードéèΣσς', 'ςσΣèéドーコニユー字漢'],
-                ['नमस्ते', 'तेस्मन'],
+                // ['नमस्ते', 'तेस्मन'], this case requires a version of intl that supports Unicode 15.1
```

Comment:
If this test case previously passed and no longer passes with the change you are making, isn't that a regression that could affect users?